.github/workflows/test.yml

```diff
@@ -1,12 +1,12 @@
 name: Run linter and tests
 
 on:
+  workflow_dispatch:
   push:
     branches:
-      - '*'
+      - main
+      - master
```

Comment:
I've just had some feedback that we should not restrict the branches. Sorry about. That.

You could do:

`on: [push, pull_request, workflow_dispatch]` 

Whatever you prefer

Reply:
Oops, just merged this. I'll fix in another #23